LICENSE

```diff
@@ -1,15 +1,202 @@
-Envoy
+                                 Apache License
```

Comment:
Nit: first line is empty in the [original Apache license file](http://www.apache.org/licenses/LICENSE-2.0.txt).